optuna_dashboard/ts/components/GraphHistory.tsx

```diff
@@ -69,6 +69,20 @@ export const GraphHistory: FC<{
       theme.palette.mode,
       markerSize
     )
+    const element = document.getElementById(plotDomId)
+    if (element != null && studies.length === 1) {
+      // @ts-ignore
+      element.on("plotly_click", function (data) {
+        const link =
+          URL_PREFIX +
+          `/studies/${studies[0].id}}/trials?numbers=${data.points[0].x}`
```

Comment:
In this [ref](https://plotly.com/javascript/plotlyjs-events/#event-data), it seems that we can get `curveNumber` from data.

How about replace `studies[0]` to `studies[Math.floor(data.points[0].curveNumber / 2)]`? In that case, it also work with `studies.length >=1` .